app/lib/actions/katello/repository/clone_docker_content.rb

```diff
@@ -2,14 +2,20 @@ module Actions
   module Katello
     module Repository
       class CloneDockerContent < Actions::Base
-        def plan(source_repo, target_repo)
+        def plan(source_repo, target_repo, filters)
+          filters = filters.docker unless filters.is_a? Array
+
+          if filters.any?
+            clause_gen = ::Katello::Util::DockerManifestClauseGenerator.new(source_repo, filters)
+            clause_gen.generate
+            copy_clauses = clause_gen.copy_clause
```

Comment:
My tests with existing code has the include/exclude filters working for docker tags. The remove_clause seems to only play a part when there are package deps being found.